### PR TITLE
Added algorithm for connect()

### PR DESCRIPTION
**Pull request by avayvod**

### CODE REVIEW

**Comment by avayvod** [comment]
@mounirlamouri PTaL

File: index.html [hunk 1]
```diff
@@ -323,6 +323,140 @@
...
+          <li>
+            Set the <a for="RemotePlayback">state</a> of the <var>remote</var> object to
+            <a for="RemotePlabyackState">connecting</a>.
+          </li>
+          <li>
+            <a>Resolve</a> <var>promise</var> with <code>true</code>.
+          </li>
+          <li>
+            <a>Queue a task</a> to <a>fire</a> a <a>trusted event</a> with
+            the name <a for="PresentationRequest">statechange</a>,
```
Comment (mounirlamouri) [inline index.html]: I think you can use `simple event`: https://www.w3.org/TR/html5/webappapis.html#fire-a-simple-event

File: index.html [hunk 2]
```diff
@@ -323,6 +323,140 @@
...
+            </ol>
+          </li>
+          <li>
+            If the user <em>denied permission</em> to use a device, resolve
+            <var>promise</var> with <code>false</code>, and abort
+            all remaining steps.
+          </li>
+          <li>
+            Otherwise, the user <em>granted permission</em> to use a device;
+            let <var>device</var> be that device and <var>userAgent</var>
```
Comment (mounirlamouri) [inline index.html]: Isn't `userAgent` the current user agent?

File: index.html [hunk 3]
```diff
@@ -323,6 +323,140 @@
...
+            The event must not bubble, must not be cancelable, and has no
+            default action.
+          </li>
+          <li>
+            If any of the following steps fails, abort all remaining steps
+            and <a>stop the remote playback</a> for <var>remote</var>.
+          </li>
+          <li>
+            Using an implementation specific mechanism, tell <var>userAgent</var>
+            to start remote playback of <var>remotePlaybackSource</var> on <var>device</var>.
```
Comment (mounirlamouri) [inline index.html]: Why isn't that: `Using an implementation specific mechanism, start the remote playback of <var>remotePlaybackSource</var> on <var>device</var>`?

**Comment by mounirlamouri** [comment]
lgtm with comments addressed.

File: index.html [hunk 4]
```diff
@@ -323,6 +323,140 @@
             is not yet established or being established.</li>
         </ul>
       </p>
+      <section>
+        <h4>
+          Initiate remote playback
+        </h4>
+        <p>
+          When the <code><dfn for="RemotePlayback">connect</dfn>()</code>
```
Comment (mounirlamouri) [inline index.html:327]: Hmm, now that `connect()` is defined, can you remove the comment in the interface definition?

File: index.html (hunk 1 above)
Comment (avayvod) [inline index.html]: Done.

File: index.html (hunk 4 above)
Comment (avayvod) [inline index.html:327]: Yep.

File: index.html (hunk 2 above)
Comment (avayvod) [inline index.html]: Yep.

File: index.html (hunk 3 above)
Comment (avayvod) [inline index.html]: Copy/paste artifact?

**Pull request merged**